demo/app/components/demo-map/demo-map.component.html

```diff
@@ -22,4 +22,5 @@
     <!--<arc-layer #layer></arc-layer>-->
     <!--<draw-on-map-layer></draw-on-map-layer>-->
     <!--<polygon-layer></polygon-layer>-->
+    <keyboard-control-layer *ngIf="appSettingsService.keyboardControl"></keyboard-control-layer>
```

Comment:
1. ac-map will have getKeyboardControl()
then you can do @ViewChild('ac-map') map; map.getKeyboardControl()...
2. maybe add it to `ViewersManagerService` its a service that holds all the viewers in the app by map id. we can add also the keyboardControl to there. (not sure about that...)

I most say I dont think there is a great solution for this, I prefer option 1.